src/jdk.compiler/share/classes/com/sun/tools/javac/comp/TypeEnter.java

```diff
@@ -1176,16 +1176,9 @@ private void addValueMembers(JCClassDecl tree, Env<AttrContext> env) {
                         switch (parameterTypes.size()) {
```

Comment:
I think this switch should be replaced with if (parameterTypes.size() == 0) - with only one case it reads off to use a switch.

Reply:
Applied comments, thanks !